### PR TITLE
Exclude locale from the sync path

### PR DESCRIPTION
**Pull request by SooluThomas**
### Summary

As mentioned in the review comment https://github.com/qiskit-community/qiskit-translations/pull/686#discussion_r669534783, rclone has to exclude locale from the sync path otherwise when machine learning goes to push an update it will delete all the contents from the locale path.

### Details and comments

This PR has a higher priority than #160.

#### Order of merging
- This PR 
- https://github.com/qiskit-community/qiskit-translations/pull/686
- #160

### CODE REVIEW

File: tools/deploy_documentation.sh [hunk 1]
```diff
@@ -26,4 +26,4 @@ pwd
 # Push to qiskit.org website
 openssl aes-256-cbc -K $encrypted_rclone_key -iv $encrypted_rclone_iv -in tools/rclone.conf.enc -out $RCLONE_CONFIG_PATH -d
 echo "Pushing built docs to website"
-rclone sync --progress ./docs/_build/html IBMCOS:qiskit-org-web-resources/documentation/machine-learning
+rclone sync --progress --exclude-from ./locale/** ./docs/_build/html IBMCOS:qiskit-org-web-resources/documentation/machine-learning
```
Comment (mtreinish) [inline tools/deploy_documentation.sh:29]: You want to use `--exclude` here, `--exclude-from` is to load a list of exclusion regexes from a file.

```suggestion
rclone sync --progress --exclude ./locale/** ./docs/_build/html IBMCOS:qiskit-org-web-resources/documentation/machine-learning
```

File: tools/deploy_documentation.sh [hunk 2]
```diff
@@ -26,4 +26,4 @@ pwd
 # Push to qiskit.org website
 openssl aes-256-cbc -K $encrypted_rclone_key -iv $encrypted_rclone_iv -in tools/rclone.conf.enc -out $RCLONE_CONFIG_PATH -d
 echo "Pushing built docs to website"
-rclone sync --progress ./docs/_build/html IBMCOS:qiskit-org-web-resources/documentation/machine-learning
+rclone sync --progress --exclude ./locale/** ./docs/_build/html IBMCOS:qiskit-org-web-resources/documentation/machine-learning
```
Comment (mtreinish) [inline tools/deploy_documentation.sh:29]: Actually looking at this did you confirm the `./` works. Looking at the exclude regex we use in qiskit/qiskit it's `locale/**`: https://github.com/Qiskit/qiskit/blob/master/tools/other-builds.txt#L1 (which is also what we used prior to the separate exclude file: https://github.com/Qiskit/qiskit/blob/453c57b348b5f0c8fe80493953bf5eb30cec3b04/tools/deploy_documentation.sh#L32

Comment (SooluThomas) [reply tools/deploy_documentation.sh:29]: I'll push a commit to use `locale/**` just to be on the safer side

Comment (SooluThomas) [reply tools/deploy_documentation.sh]: Fixed in https://github.com/Qiskit/qiskit-machine-learning/pull/165/commits/d81d4845e034ca6e71d0cd6258222a4acc772e51

**Review by mtreinish — APPROVED** [review]

**Review by manoelmarques — APPROVED** [review]

**Pull request merged**